src/net_processing.cpp

```diff
@@ -751,7 +751,7 @@ class PeerManagerImpl final : public PeerManager
     int nSyncStarted GUARDED_BY(cs_main) = 0;
 
     /** Hash of the last block we received via INV */
-    uint256 m_last_block_inv_triggering_headers_sync{};
+    uint256 m_last_block_inv_triggering_headers_sync GUARDED_BY(g_msgproc_mutex){};
```

Comment:
In the previous incarnation of this PR the commit `net_processing: add thread safety annotations for PeerManagerImpl members accessed only via the msgproc thread` contained just this, 1-line, hunk.

Now it contains many hunks that change `NetEventsInterface::g_msgproc_mutex` to `g_msgproc_mutex`. Better to use `g_msgproc_mutex` initially and not have to modify it in a subsequent commit.

Reply:
Ugh, yes. Fixed